typescriptSrc/animator.ts

```diff
@@ -14,6 +15,7 @@ import animatorHelpers = require('./animatorHelpers');
 import assert = require( './assert' );
 import backtracking = require( './backtracking' ) ;
 import collections = require( './collections' );
+import createHTMLElements = require('./createHTMLElements');
```

Comment:
This line is stopping typescript from compiling on my machine. The file is named createHtmlElements.ts, not createHTMLElements.ts